pkg/cmd/admin/router/router.go

```diff
@@ -656,6 +665,13 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 			exposedPorts[i].HostPort = 0
 		}
 	}
+	ResourceRequirements, err := kubectl.HandleResourceRequirements(map[string]string{"limits": cfg.ResourceLimits, "requests": cfg.ResourceRequests})
+	if err != nil {
+		return fmt.Errorf("limits or requests could not be interpreted: %v", err)
+	}
+	if cfg.ResourceLimits == "" || cfg.ResourceRequests == "" {
+		glog.Warningf("Running %s without resource requests or limits is bad practice in a production environment", cfg.Name)
```

Comment:
The reality is we need to check that both memory and cpu have made an explicit request.  Absent either resource, the pod is still 'BestEffort' qos and will be first to get the boot if a node finds itself out of resource and has eviction enabled (kube 1.3 feature).  I think the warning can be updated to state: "Running %s with BestEffort quality of service is not recommended in a production environment.  A resource request for cpu and memory is strongly recommended."